doc/api/child_process.markdown

```diff
@@ -58,31 +61,33 @@ stalling the event loop while spawned processes complete.
 
 ## Asynchronous Process Creation
 
-The `child_process.spawn()`, `child_process.fork()`, `child_process.exec()`,
-and `child_process.execFile()` methods all follow the idiomatic asynchronous
-programming pattern typical of other Node.js APIs.
+The [`child_process.spawn()`][], [`child_process.fork()`][],
+[`child_process.exec()`][], and [`child_process.execFile()`][] methods all
+follow the idiomatic asynchronous programming pattern typical of other Node.js
+APIs.
 
 Each of the methods returns a [`ChildProcess`][] instance. These objects
 implement the Node.js [`EventEmitter`][] API, allowing the parent process to
 register listener functions that are called when certain events occur during
 the life cycle of the child process.
 
-The `child_process.exec()` and `child_process.execFile()` methods additionally
-allow for an optional `callback` function to be specified that is invoked
-when the child process terminates.
+The [`child_process.exec()`][] and [`child_process.execFile()`][] methods
+additionally allow for an optional `callback` function to be specified that is
+invoked when the child process terminates.
 
 ### Spawning `.bat` and `.cmd` files on Windows
 
-The importance of the distinction between `child_process.exec()` and
-`child_process.execFile()` can vary based on platform. On Unix-type operating
-systems (Unix, Linux, OSX) `child_process.execFile()` can be more efficient
-because it does not spawn a shell. On Windows, however, `.bat` and `.cmd`
-files are not executable on their own without a terminal, and therefore cannot
-be launched using `child_process.execFile()`. When running on Windows, `.bat`
-and `.cmd` files can be invoked using `child_process.spawn()` with the `shell`
-option set, with `child_process.exec()`, or by spawning `cmd.exe` and passing
-the `.bat` or `.cmd` file as an argument (which is what the `shell` option and
-`child_process.exec()` do).
+The importance of the distinction between [`child_process.exec()`][] and
+[`child_process.execFile()`][] can vary based on platform. On Unix-type
+operating systems (Unix, Linux, OSX) [`child_process.execFile()`][] can be more
+efficient because it does not spawn a shell. On Windows, however, `.bat` and
+`.cmd` files are not executable on their own without a terminal, and therefore
+cannot be launched using [`child_process.execFile()`][]. When running on
+Windows, `.bat` and `.cmd` files can be invoked using
+[`child_process.spawn()`][] with the `shell` option set, with
+[`child_process.exec()`][], or by spawning `cmd.exe` and passing the `.bat` or
+`.cmd` file as an argument (which is what the `shell` option and
+[`child_process.exec()`][] do).
```

Comment:
Its basically impossible to tell during review without an eyeball straining line-by-line review whether you have just added markdown links to the above, or whether you also made some changes to the docs.

The markup changes can get rubber stamped if they look OK after converting to html, text changes need closer review.